galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -257,55 +256,15 @@ def copy_content(self, request, *args, **kwargs):
 
         collection_version = self.get_collection_version()
         src_repo, dest_repo = self.get_repos()
-        source_pks = src_repo.content.values_list("pk", flat=True)
-        content_types = src_repo.content.values_list('pulp_type', flat=True).distinct()
-        cv_pk = collection_version.pk
-
-        content = [cv_pk]
-
-        # collection signatures
-        if 'ansible.collection_signature' in content_types:
-            signatures_pks = CollectionVersionSignature.objects.filter(
-                signed_collection=cv_pk,
-                pk__in=source_pks
-            ).values_list("pk", flat=True)
-            if signatures_pks:
-                content.append(*signatures_pks)
-
-        # collection version mark
-        if 'ansible.collection_mark' in content_types:
-            marks_pks = CollectionVersionMark.objects.filter(
-                marked_collection=cv_pk,
-                pk__in=source_pks
-            ).values_list("pk", flat=True)
-            if marks_pks:
-                content.append(*marks_pks)
-
-        # collection deprecation
-        if 'ansible.collection_deprecation' in content_types:
-            deprecations_pks = AnsibleCollectionDeprecated.objects.filter(
-                pk__in=source_pks,
-                namespace=collection_version.namespace,
-                name=collection_version.name,
-            ).values_list("pk", flat=True)
-            if deprecations_pks:
-                content.append(*deprecations_pks)
```

Comment:
append() takes one argument, how did I miss that in the previous review 🤦🏻

Reply:
ooops, sorry :facepalm: :face_with_spiral_eyes: